openlibrary/plugins/openlibrary/js/realtime_account_validation.js

```diff
@@ -71,18 +77,22 @@ export function initRealTimeValidation() {
     function validateEmail() {
         const value_email = $(this).val();
         if (value_email !== '') {
-            $.ajax({
-                url: '/account/validate',
-                data: { email: value_email },
-                type: 'GET',
-                success: function(errors) {
-                    if (errors.email) {
-                        renderError('#emailAddr', '#emailAddrMessage', errors.email);
-                    } else {
-                        clearError('#emailAddr', '#emailAddrMessage');
+            if (!(/.*@.*\..*/.test(value_email))) {
```

Comment:
This regex should be sufficient for the common case, though it will over accept input

e.g. `(/.*@.*\..*/).test("mek@@@archive.org")`

Reply:
Good point! I can update the regex in `forms.py` as well to make both of them a little more accurate.